index.js

```diff
@@ -24,6 +24,14 @@ var repl = readline.createInterface({
   }
 });
 
+// exit on SIGINT, give a timestamp
+//
+repl.on('SIGINT', function(){
+  process.stdout.write('\n');
```

Comment:
Normally when I do Ctrl+C the prompt is empty so yes. If you don't like the date you could do a PR.